buildstockbatch/postprocessing.py

```diff
@@ -251,22 +254,41 @@ def merge_schema_dicts(dict1, dict2):
     return new_dict
 
 
-def read_enduse_timeseries_parquet(fs, filename, all_cols):
+def read_enduse_timeseries_parquet(fs, all_cols, filename):
     with fs.open(filename, 'rb') as f:
         df = pd.read_parquet(f, engine='pyarrow')
     building_id = int(re.search(r'bldg(\d+).parquet', filename).group(1))
     df['building_id'] = building_id
     for col in set(all_cols).difference(df.columns.values):
         df[col] = np.nan
-    return df[all_cols]
+    df = df[all_cols]
+    df.set_index('building_id', inplace=True)
+    return df
+
+
+def concat_and_normalize(fs, all_cols, src_path, dst_path, partition_columns, indx, bldg_ids, partition_vals):
+    dfs = []
+    for bldg_id in sorted(bldg_ids):
+        src_filename = Path(src_path) / f"bldg{bldg_id:07}.parquet"
+        df = pd.read_parquet(src_filename, engine='pyarrow')
```

Comment:
Should use `fs`, something like this.

```suggestion
        with fs.open(src_filename, 'rb') as f:
            df = pd.read_parquet(f, engine='pyarrow')
```